deploy/demoapp/templates/twitter-cass-user.yaml

```diff
@@ -19,11 +19,29 @@ spec:
       - name: twitter-cass-user
         image: {{ .Values.image.repository }}/twitter-cass-user:{{ .Values.image.tag }}
         imagePullPolicy: {{ .Values.image.pullPolicy }}
+{{- if .Values.grpc.max_connection_age_ms }}
+        args:
+        - --grpc-max-connection-age-ms {{ .Values.grpc.max_connection_age_ms }}
+{{- end }}
         ports:
           - containerPort: 50051
         env:
         - name: CASSANDRA_HOST_ADDRESS
           value: cassandra
+{{- if .Values.grpc.debug }}
+        - name: GRPC_VERBOSITY
+          value: DEBUG
+{{- if .Values.grpc.trace }}
+        - name: GRPC_TRACE
+          value: {{ .Values.grpc.trace | quote }}
+{{- end }}
+{{- end }}
+{{- if .Values.instana.enabled }}
```

Comment:
not really an issue, but what happens if this is not changed in the app?
I thought instana is supposed to dynamically inject instrumentations without having to change the app/deployment

Reply:
@esara , this is not about `Instana` instrumentation, this is about configuring network access for the instrumentation, such that applications running inside a kubernetes container can locate the agent and report metrics. 

Please see [Configure Network Access for Monitored Applications](https://www.instana.com/docs/setup_and_manage/host_agent/on/kubernetes/#configure-network-access-for-monitored-applications).